csharp.html.markdown

```diff
@@ -869,6 +869,49 @@ on a new line! ""Wow!"", the masses cried";
 
         public DbSet<Bicycle> Bikes { get; set; }
     }
+    
+    // New C# 6 features
+    class GlassBall : IJumpable, IBreakable
+    {
+        // Autoproperty initialisers
```

Comment:
the document uses "initialize" previously. either change it, or change these ones :)